README.md

```diff
@@ -98,23 +87,39 @@ make install
 | ------------------------------ | ----------------- | ---------------------------------------------------------------------------------------------------- |
-| BUILD_DEBUG_WASM               | `ON`,`OFF`        | Built wasms should include debug information                                                         |
+| BUILD_DEBUG_WASM               | `yes`,`no`        | Build debug wasms (Needed for using the debugger)                                                    |
 | BUILD_DOC                      | `yes`,`no`        | Build the technical documentation                                                                    |
 | BUILD_JS                       | `yes`,`no`        | Build application front-ends                                                                         |
 | BUILD_NATIVE                   | `yes`,`no`        | Build the native programs (`psinode`, `psitest`)                                                     |
+| BUILD_RELEASE_WASM             | `yes`,`no`        | Build optimized release wasms                                                                        |
 | BUILD_RUST                     | `yes`,`no`        | Build `psibase` CLI tool                                                                             |
-| CMAKE_BUILD_TYPE               | `Debug`,`Release` | Determines level of optimization                                                                     |
-| CMAKE_EXPORT_COMPILE_COMMANDS  | `ON`,`OFF`        | Tell CMake to generate compile_commands.json, used by various tools (e.g. clang format intellisense) |
+| CMAKE_BUILD_TYPE               | `Debug`,`Release` | Determines level of optimization for the native programs                                             |
+| CMAKE_EXPORT_COMPILE_COMMANDS  | `yes`,`no`        | Tell CMake to generate compile_commands.json, used by various tools (e.g. clang format intellisense) |
 | CMAKE_INSTALL_PREFIX           | `psidk`           | Binary install location.                                                                             |
 
+## Running psibase infrastructure 🚀
+
+> 🔍 We currently do not provide an ARM binary or docker image. If your device uses an ARM chip, you should build psinode (or its [docker image](https://github.com/gofractally/image-builders)) yourself. 
+
+To run psinode, you can either build from source as described above, or you can install prebuilt binaries through one of the following two methods:
+1. Installing a binary release, the latest of which can be found on our [Releases](https://github.com/gofractally/psibase/releases) page. We provide binary packages for both Ubuntu 20.04 and Ubuntu 22.04.
+2. Installing our psinode [Docker Image](https://github.com/orgs/gofractally/packages/container/package/psinode).
+
+To execute psinode from the docker image, you can use something the following docker command (make sure to specify the `VERSION` you want to run):
```

Comment:
"something the following" doesn't sound right

Reply:
fixed